index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="UTF-8">
   <title>Fork Me! FCC: Test Suite Template</title>
-  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="style.css">
 </head>
 <body>
   <h1>Hello Microverse</h1>
```

Comment:
- [ ] [REQUIRED] Please, make sure to add a class to the `h1` tag as requested in project requirements. And then use that class to add the color to separate style from structure.